packages/infra/lib/api-stack.ts

```diff
@@ -1494,6 +1504,37 @@ export class APIStack extends Stack {
     return fhirToMedicalRecordLambda;
   }
 
+  private setupFhirToCdaConverterLambda(ownProps: {
```

Comment:
Let's move this to the LambdaNestedStack, please. We should be very strict from now on with adding new resources to the APIStack.

Being introduced in this PR and downstream ones: https://github.com/metriport/metriport/pull/2018

Reply:
Sounds good. Done